Introduce SQL migrations (#907)
kubearchive/kubearchive: docs/modules/ROOT/pages/getting-started/installation.adoc

== Prepare the Database

. Create a user and a password for KubeArchive to use.

> . Create a database for KubeArchive, with admin permissions granted to the user you just created.

Reviewer: ```suggestion
. Create a database for KubeArchive, with admin permissions granted to new user.
```

>   Configure its timezone as UTC and its locale as UTF-8.

Author: Setup is not executed by the user, setup is replicating what a user wanting to install KubeArchive should do.

Maybe the timezone is not really important, but normally distributed systems are set to UTC.
